src/engine/Server.cpp

```diff
@@ -352,7 +357,7 @@ string Server::composeResponseJson(const ParsedQuery& query,
   os << ",\n";
 
   os << "\"time\": {\n"
-     << "\"total\": \"" << _requestProcessingTimer.usecs() / 1000.0 << "ms\",\n"
+     << "\"total\": \"" << _requestProcessingTimer.msecs() / 1000.0 << "ms\",\n"
```

Comment:
This seems wrong, if `Timer::msecs()` gives you milliseconds you shouldn't have to divide by 1000.0 to get "ms"